multi-watch/settingmanager.cpp

```diff
@@ -0,0 +1,46 @@
+#include "settingmanager.h"
+
+SettingManager::SettingManager()
+{
+
+}
+
+void SettingManager::saveTabOrder(QTabWidget *tabWidget)
+{
+  QSettings settings("multiWatchSetting", "multiWatch");
+  int nTab = tabWidget->count();
+  QString tabOrderTxt = "";
+  int i;
+  tabOrderTxt.append(tabWidget->tabText(0));
+  for(i = 1; i< nTab; i++){
+      tabOrderTxt.append("_");
+      tabOrderTxt.append(tabWidget->tabText(i));
+    }
+  qDebug() << "saving "<< tabOrderTxt;
+  settings.setValue("TabOrder", tabOrderTxt);
```

Comment:
If this `TabOrder` is only used in this class then make in class constant;
If this is shared among several classes, make a separate class to hold these kinds of values or you can use `#define` lke QT does for these kind of thinges